test/github.ts

```diff
@@ -763,14 +845,10 @@ describe('GitHub', () => {
         sha: 'abc123',
         notes: 'Some release notes',
       });
-      await assert.rejects(promise, error => {
-        return (
-          error instanceof DuplicateReleaseError &&
-          // ensure stack contains calling method
-          error.stack?.includes('GitHub.createRelease') &&
-          !!error.cause
-        );
-      });
```

Comment:
This is a bit sad, error stacks do not include the `src/github.js` context anymore when using octokit plugins. That may result in error logs more difficult to analyze, I think we should see in practice if that's a problem or not.